src/components/row/row.js

```diff
@@ -4,84 +4,38 @@ import classNames from 'classnames';
 import Column from './column';
 import './row.scss';
 
-/**
- * A row widget.
- *
- * This is a standalone row widget used for layout; for table rows use the table-row widget.
- *
- * == How to use a Row in a component:
- *
- * In your file
- *
- *   import { Row, Column } from 'carbon-react/lib/components/row';
- *
- * To render the Row:
- *
- *   <Row>
- *     <Column>Column1</Column>
- *     <Column>Column2</Column>
- *   </Row>
- *
- * A Rows child must be of type Column
- *
- * @class Row
- * @constructor
- */
 class Row extends React.Component {
   static propTypes = {
-
     /**
-     * The elements to be rendered in the row
-     *
-     * @property children
-     * @type {Object | Array}
+     * This component supports children of type Column.
      */
-    children: PropTypes.oneOfType([
-      PropTypes.array,
-      PropTypes.object
-    ]),
+    children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
```

Comment:
preferred this over multiple lines as it was

Reply:
done